plants-backend-adapter-service/src/test/resources/commodity/PlantProducts.json

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
Is this being used anywhere? I thought we are using one combined schema for all commodities?